be/test/util/threadpool_test.cpp

```diff
@@ -328,15 +328,9 @@ TEST_F(ThreadPoolTest, TestZeroQueueSize) {
 TEST_F(ThreadPoolTest, TestDeadlocks) {
     ::testing::FLAGS_gtest_death_test_style = "threadsafe";
 #ifdef __APPLE__
-    const char* death_msg =
-            "_ZNSt3__1L8__invokeIRNS_6__bindIMN5doris10ThreadPoolEFvvEJPS3_EEEJEEEDTclscT_fp_"
-            "spscT0_fp0_EEOS9_DpOSA_|_ZNSt3__18__invokeB6v15007IRNS_6__"
-            "bindIMN5doris10ThreadPoolEFvvEJPS3_EEEJEEEDTclclsr3stdE7declvalIT_"
-            "EEspclsr3stdE7declvalIT0_EEEEOS9_DpOSA_";
+    const char* death_msg = "ThreadPool";
 #else
-    const char* death_msg =
-            "_ZNSt5_BindIFMN5doris10ThreadPoolEFvvEPS1_EE6__callIvJEJLm0EEEET_OSt5tupleIJDpT0_"
-            "EESt12_Index_tupleIJXspT1_EEE";
+    const char* death_msg = "ThreadPool";
```

Comment:
I don't think this modification is suitable. If we run BE UT whose build type isn't release, we can check the stack frames more accurately.

Reply:
Fixed. but AFAIC, this testcase is simple and clear enough to assert that the only possibility that deadlock occurs is what we expect(in ThreadPool). actually the msg origin is just to check the "call from ThreadPool" too.